src/pages/RegisterPage/RegisterPage.tsx

```
@@ -486,12 +577,29 @@ const RegisterPage = () => {
-                {submitError}
-              </p>
-            ) : null}
+            {/* Error Messages Display */}
+            {fieldErrors.general && (
+              <div className="rounded-lg border-l-4 border-red-500 bg-red-50 p-4 shadow-sm">
+                <div className="flex items-start">
+                  <svg
+                    className="h-5 w-5 flex-shrink-0 text-red-500"
+                    fill="currentColor"
+                    viewBox="0 0 20 20"
+                  >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)